scripts/analytics/node_log_events.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from modularodm import Q
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+
+from website.app import init_app
+from website.project.model import NodeLog
+from scripts.analytics.base import EventAnalytics
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class NodeLogEvents(EventAnalytics):
+
+    @property
+    def collection_name(self):
+        return 'node_log_events'
+
+    def get_events(self, date):
+        """ Get all node logs from a given date for a 24 hour period,
+        ending at the date given.
+        """
+        super(NodeLogEvents, self).get_events(date)
+
+        # In the end, turn the date back into a datetime at midnight for queries
+        date = datetime(date.year, date.month, date.day)
+
+        logger.info('Gathering node logs between {} and {}'.format(
+            date, (date + timedelta(1)).isoformat()
+        ))
+
+        node_log_query = Q('date', 'lt', date + timedelta(1)) & Q('date', 'gte', date)
+
+        node_logs = NodeLog.find(node_log_query)
+        node_log_events = []
+        for node_log in node_logs:
+            event = {
+                'keen': {'timestamp': node_log.date.isoformat()},
+                'date': node_log.date.isoformat(),
```

Comment:
❗️ `node_log.date` is a naive datetime, so `.isoformat()` doesn't append any TZ information to it.  `keen.timestamp` (correctly) assumes we mean UTC, but doesn't do the same for `date`:
![screen shot 2016-11-06 at 10 30 00 am](https://cloud.githubusercontent.com/assets/86835/20039096/469755b6-a40c-11e6-86f4-f21e1d4c4aa3.png)

Can you add the timezone so that it's explicit in Keen?